action.yml

```diff
@@ -6,8 +6,8 @@ inputs:
     required: false
     default: 'https://cloud.gatling.io'
   api_token:
-    description: 'A Gatling Enterprise API token with at least Start permission (we recommend storing it securely using GitHub''s encrypted secrets; see https://docs.github.com/en/actions/security-guides/encrypted-secrets#using-encrypted-secrets-in-a-workflow).'
-    required: true
+    description: 'A Gatling Enterprise API token with at least Start permission (we recommend storing it securely using GitHub''s encrypted secrets; see https://docs.github.com/en/actions/security-guides/encrypted-secrets#using-encrypted-secrets-in-a-workflow). Alternatively, you can define your token in an environment variable named GATLING_ENTERPRISE_API_TOKEN.'
```

Comment:
Is it really a good practice?

Reply:
We wanted to allow using the same env var to configure both a build step with our build plugins, and a run step with this action. I don't see any specific issue with this (at the end of the day, inputs are also passed to the action as env vars, just with a special naming convention; and both can be configured using secrets)?